apps/acceptance/lib/view.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var Marionette = require('marionette-client');
+
+function View(client) {
+    this.client = client.scope({searchTimeout: 5000});
+    this.actions = new Marionette.Actions(this.client);
+}
+
+View.prototype = {
+    //Get the root element of the app and return it.
+    getRootElement: function (rootSelector) {
+        return this.client.findElement(rootSelector);
+    },
+    //From the root, find a child element based on selector.
+    findElement: function (selectorObject) {
+        return this.getRootElement(selectorObject.root).findElement(selectorObject.element);
+    },
+    //Return whether or not the root element is displayed.
+    rootDisplayed: function () {
```

Comment:
This function is not currently used, I would remove it, and add it when we need it.